package.json

```
@@ -30,11 +31,24 @@
     "eslint-plugin-jsx-a11y": "^6.0.3",
     "eslint-plugin-prettier": "^3.1.2",
     "eslint-plugin-react": "^7.9.1",
-    "prettier": "^1.16.4",
-    "jest": "^23.1.0"
+    "husky": ">=4",
+    "jest": "^23.1.0",
+    "lint-staged": ">=10",
```

Might want to specify compatible versions (only minor & patch). Or just a specific version.

```suggestion
    "lint-staged": "^10",
```